src/test/ui/consts/const-eval/promoted_errors.rs

```diff
@@ -1,23 +1,28 @@
+// revisions: default noopt opt opt_with_overflow_checks
+//[noopt]compile-flags: -C opt-level=0
+//[opt]compile-flags: -O
+//[opt_with_overflow_checks]compile-flags: -C overflow-checks=on -O
+
 // build-pass
 // ignore-pass (emit codegen-time warnings and verify that they are indeed warnings and not errors)
-// compile-flags: -O
 
-#![warn(const_err)]
+#![warn(const_err, overflow, panic)]
 
 fn main() {
     println!("{}", 0u32 - 1);
+    //[opt_with_overflow_checks,noopt]~^ WARN [overflow]
```

Comment:
This makes it look like `default` is the same as `opt`, which I find surprising.